.github/workflows/seqera_docs_changelog.yml

```
@@ -0,0 +1,60 @@
+name: Push changelog to Seqera Docs
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      release_name:
+        description: 'Release version (e.g. 1.0.0)'
+        required: true
+      release_body:
+        description: 'Release changelog content'
+        required: true
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+          
+      - name: Clone seqeralabs/docs
+        run: |
+          git clone https://github.com/seqeralabs/docs.git seqeralabs-docs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create changelog file
+        run: |
+          mkdir -p seqeralabs-docs/changelog/wave
+          cat << EOF > seqeralabs-docs/changelog/wave/${{ github.event.release.name || inputs.release_name }}.mdx
+          ---
+          title: Wave ${{ github.event.release.name || inputs.release_name }}
+          date: $(date +%Y-%m-%d)
+          tags: [wave]
+          ---
+
+          ${{ github.event.release.body || inputs.release_body }}
+          EOF
+
+      - uses: actions/create-github-app-token@v1
+        id: generate-token
+        with:
+          app-id: ${{ secrets.DOCS_BOT_APP_ID }}
+          private-key: ${{ secrets.DOCS_BOT_APP_PRIVATE_KEY }}
+          owner: seqeralabs
+          repositories: docs
+        
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v7
+        with:
+          branch-token: ${{ steps.generate-token.outputs.token }}
```

```suggestion
          token: ${{ steps.generate-token.outputs.token }}
          branch-token: ${{ steps.generate-token.outputs.token }}
```

https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#branch-token

```
The action first creates a branch, and then creates a pull request for the branch. For some rare use cases it can be useful, or even necessary, to use different tokens for these operations. It is not advisable to use this input unless you know you need to.
```

This action by default uses the token in the context of the action which I don't believe will have permission to create a pull request against the upstream repository.